packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -295,22 +295,42 @@ class RenderEditable extends RenderBox {
   ValueListenable<bool> get selectionEndInViewport => _selectionEndInViewport;
   final ValueNotifier<bool> _selectionEndInViewport = ValueNotifier<bool>(true);
 
+  // Returns a bool indicating if the offset is within factor of being contained
+  // within the rect.
+  static bool _containsFuzzy(Rect rect, Offset offset, [double factor = 0.1]) {
+    return rect.contains(offset)
```

Comment:
This expression could just be `rect.inflate(factor).contains(offset)`.  Suggest renaming factor => slop (here and elsewhere). We actually use "slop" in other similar names.